lib/os/Kconfig

```diff
@@ -72,4 +72,12 @@ config PRINTK_SYNC
 	  interleaving with concurrent usage from another CPU or an
 	  preempting interrupt.
 
+config TIMEUTIL_REALTIME
+	bool "Support conversion between local and civil time"
+	help
+	  When true API is provided that enables applications to read
+	  the current time based on maintained offsets and skews
+	  between the local tick clock and an external time source,
+	  possibly an RTC.
+
```

Comment:
What is the rationale behind the `CONFIG_TIMEUTIL_REALTIME` naming? "Real time" may be confusing here (Zephyr is a Real-Time Operating System).

Have you considered `CONFIG_TIMEUTIL_CIVIL_TIME`? Same goes for the naming of the API functions.

Reply:
Yes, but the [consensus interpretation of "civil time"](https://en.wikipedia.org/wiki/Civil_time) is the local time, which is almost always UTC plus an offset (that generally changes twice a year).

I'm not entirely bound to that decision.  Options I've considered include:
* civil time (implies a local time zone time scale)
* calendar time (implies a representation, not a time scale)
* wall clock
* timescale
* real time

none of which are wonderful.  I went with `REALTIME` because it's what `CLOCK_REALTIME` on a POSIX system gives you, which is UTC, and I think that the most likely use of these functions will be to produce something that approximates or provides a capability similar to `CLOCK_REALTIME`.  So: `timeutil_realtime_`.

If "realtime" is too confusing because Zephyr is a "real-time operating system" (it's not), then we can come up with a consensus alternative. 

 After reviewing all this, I'm warming to "timescale", which doesn't imply anything that's false or misleading.

Reply:
I think it is confusing because realtime term is subjective, from the designation alone, everyone could understand something different about what for example `timeutil_realtime_get()` does. Maybe CONFIG_TIMEUTIL_CTS/`timeutil_cts_get()`, for civil time scale (https://github.com/zephyrproject-rtos/zephyr/pull/28977/commits/144c887c94846ee92f1338dfeece6288d30c1ff3#diff-ca1507f5fcb1d2e792a8c80442ab64038f0e5d9939a38e6015b1f1b5d28a8431R34) ?

Reply:
CTS in Bluetooth is "Current Time Service", which is one of the potential sources for "real" time data.  So I think that spelling would be confusing.

Reply:
It is less confusing than REALTIME, at least for me :-)